src/components/modals/SetFramesFromTaskTypePreviewsModal.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+  <div
+    :class="{
+      modal: true,
+      'is-active': active
+    }"
+  >
+    <div class="modal-background" @click="$emit('cancel')"></div>
+
+    <div class="modal-content">
+      <div class="box content">
+        <h1 class="title">
+          {{ $t('shots.get_frames_from_previews') }}
+        </h1>
+
+        <p class="description">
+          {{ $t('shots.get_frames_from_previews_description') }}
+        </p>
+
+        <combobox-task-type
+          :task-type-list="productionShotTaskTypes"
+          :placeholder="$t('task_types.select_task_type')"
+          add-placeholder
+          v-model="taskTypeId"
+        />
+
+        <modal-footer
+          :error-text="$t('shots.get_frames_from_previews_error')"
+          :is-error="isError"
+          :is-loading="isLoading"
+          :is-disabled="!isFormFilled"
+          @confirm="confirm"
+          @cancel="$emit('cancel')"
+        />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { mapGetters, mapActions } from 'vuex'
+import { modalMixin } from '@/components/modals/base_modal'
+
+import ComboboxTaskType from '@/components/widgets/ComboboxTaskType'
+import ModalFooter from '@/components/modals/ModalFooter'
+
+export default {
+  name: 'get-frames-from-previews-modal',
```

Comment:
wrong component name

```suggestion
  name: 'set-frames-from-task-type-previews-modal',
```